docs/multisigs/committees.md

```diff
@@ -125,7 +125,7 @@ The full list of related contracts is available [here](https://docs.lido.fi/depl
 
 **Snapshot:** [Proposal to form TRP Committee](https://snapshot.org/#/lido-snapshot.eth/proposal/0xc00b48275e268f26b6cebf82322f281a44acaf679f6381dd612e278174671daf)
 
-**Aragon:** [Omnibus Vote #152 to set up LDO Easy Track](https://vote.lido.fi/vote/152)
+**Aragon:** [Omnibus vote #152 to set up LDO Easy Track](https://vote.lido.fi/vote/152)
```

Comment:
Missing line break?

Reply:
It was decided to remove line breaks in cases where a single link is specified for a forum post, a Snapshot, and an Aragon.